bokeh/util/serialization.py

```diff
@@ -253,9 +254,11 @@ def make_id():
     if settings.simple_ids():
         with _simple_id_lock:
             _simple_id += 1
-            return str(_simple_id)
+            str_id = str(_simple_id)
     else:
-        return make_globally_unique_id()
+        str_id = make_globally_unique_id()
+    # add bk- prefix to make id valid for css
```

Comment:
nits: 
* I personally prefer a blank line above a line comment
* can you add a link to either the issue or the CSS spec in the comment?

Reply:
Sounds good, done!